controllers/admin/AdminStoresController.php

```diff
@@ -566,6 +566,17 @@ public function beforeUpdateOptions()
         }
     }
 
+    public function updateOptionPsShopName($value)
```

Comment:
It would be nice to have the shop ID as an optional parameter and use the context shop id if it's not specified. 👍